brave/src/main/java/brave/internal/Platform.java

```diff
@@ -178,6 +177,10 @@ static Jre7 buildIfSupported() {
       return null;
     }
 
+    @IgnoreJRERequirement @Override public String getHostString(InetSocketAddress socket) {
```

Comment:
subtle, but socket.getHostString was added in java 7. We still support lowest java level. For example, netty and dubbo are Java 6+.